tests/integration_tests/functional/test_mmds.py

```diff
@@ -13,6 +17,21 @@ def _assert_out(stdout, stderr, expected):
     assert stdout.read() == expected
 
 
+def _metadata_json(test_microvm, metadata_file):
```

Comment:
Does it make sense to add this functionality in framework/microvm.py ?

Reply:
It does, yes. It's a microVM related property and it could be used in other tests too.